docs/source/en/autoclass_tutorial.mdx

```diff
@@ -23,6 +23,7 @@ Remember, architecture refers to the skeleton of the model and checkpoints are t
 In this tutorial, learn to:
 
 * Load a pretrained tokenizer.
+* Load an image processor
```

Comment:
Guess you deliberately removed `pretrained`? No strong opinion, just not super sure if we prefer to keep it as the same as with other entries here.

Reply:
Nope. I'll add it back. No image processors are 'pretrained' - but they are preconfigured and it's possible that there's some learnt element added to them in the future.